matterpy/contrib/rss.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+import asyncio
+import feedparser
+from collections import defaultdict
+
+_conf = None
+_mgr = None
+
+
+def init(manager, conf):
+    global _conf
+    global _mgr
+
+    _conf = conf
+    _mgr = manager
+
+    loop = asyncio.get_event_loop()
+    feedsdict = defaultdict(dict)
+
+    for key in _conf:
+        if '.' in key:
+            keys = key.split('.')
+            feedsdict[keys[1]][keys[2]] = _conf.get(key)
+    for key in feedsdict:
+        loop.create_task(post_rss(feedsdict.get(key)))
+
+
+async def post_rss(feedinfo):
+    entry = 0
+    last_message = ''
+    feed = feedparser.parse(feedinfo.get('url'))
+    while True:
+        try:
+            for i in feed.entries:
+                feed = feedparser.parse(feedinfo.get('url'))
+                message = post_to_text(feed, entry, feedinfo.get('format'))
+                if last_message < feed.entries[entry].published:
```

Comment:
The variable `entry` is never counted up...

Why not use `for entry in feed.entries` a few lines above and get rid of the `feed.entries[entry]` code bit?